units/cloud_cleanup.go

```diff
@@ -48,6 +50,7 @@ func makeCloudCleanupNameJob() *cloudCleanupJob {
 func NewCloudCleanupJob(env evergreen.Environment, ts, provider, region string) amboy.Job {
 	j := makeCloudCleanupNameJob()
 	j.SetID(fmt.Sprintf("%s.%s.%s.%s", cloudCleanupName, provider, region, ts))
+	j.SetScopes([]string{fmt.Sprintf("%s.%s.%s", cloudCleanupName, provider, region)})
```

Comment:
This should also use `j.SetEnqueueScopes(<the scope>)`.

Reply:
I'm understanding what you're saying that we don't want it to run a bunch of times if the queue gets backed up and then clears. 
Is that what you meant?

I used `j.SetEnqueueAllScopes(true)`. I think that's equivalent to your suggestion. Is that correct?

Reply:
Yes, your understanding is correct.

Yes, the usage is correct since there's only 1 scope in this case.